packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/bootstrapping.integtest.ts

```diff
@@ -17,7 +17,7 @@ integTest('can bootstrap without execution', withDefaultFixture(async (fixture)
   });
 
   expect(resp.Stacks?.[0].StackStatus).toEqual('REVIEW_IN_PROGRESS');
-}));
+}, false));
```

Comment:
This is kind of ugly. Can we not add something like:

```ts
export interface DisableBootstrapContext {
  readonly bootstrapDisabled?: boolean;
}

export function withoutBootstrap<A extends object>(block: (context: A & DisableBootstrapContext) => void) {
  return () => block({ ...context, bootstrapDisabled: true });
}
```

And have `withCdkApp` check the `bootstrapDisabled` flag?

Reply:
updated!